wazo_confd/plugins/ingress_http/api.yml

```diff
@@ -102,6 +103,13 @@ parameters:
     type: string
     name: http_ingress_uuid
     in: path
+  fallback_to_master:
```

Comment:
We maybe want to be consistent with our logic inside confd and use same logic than `user_external_app` (even if logic is user fallbacking on tenant config)
```
?view=fallback
```

side note: you document `fallback_to_master` but you implement `fallback`

Reply:
for the doc - Yes my bad, read too fast